var/spack/repos/builtin/packages/cube/package.py

```diff
@@ -36,6 +36,11 @@ class Cube(Package):
     homepage = "http://www.scalasca.org/software/cube-4.x/download.html"
     url = "http://apps.fz-juelich.de/scalasca/releases/cube/4.2/dist/cube-4.2.3.tar.gz"
 
+    # Some versions fail with parallel builds/installs
+    parallel = False
+
+    version('4.3.5', 'e5dce986e3c6381ea3a5fcb66c553adc')
+
     version('4.3.4', '50f73060f55311cb12c5b3cb354d59fa',
             url='http://apps.fz-juelich.de/scalasca/releases/cube/4.3/dist/cube-4.3.4.tar.gz')
```

Comment:
Now that you added a `url_for_version`, can you get rid of these version-specific URLs? You can test to see if things are working correctly by running `spack info`.